setup.cfg

```diff
@@ -4,7 +4,14 @@ summary = Molecule aids in the development and testing of Ansible roles.
 license = MIT
 description-file = README.rst
 home-page = https://github.com/ansible/molecule
-requires-python = 2.7
+project_urls =
```

Comment:
Is this meant to be in this PR?

Reply:
It depends. I wanted to add a link to Travis CI to metadata. So it's kinda related. But this can wait, of course. We're not publishing a new dist right now anyway. I think I'll go ahead and divide this big PRs into smaller ones...

Reply:
I agree, smaller PRs will be easier to review and merge. This PR has grown quite a lot.

Reply:
Yeah, that's because I'm discovering new issues over time and the only way to test fixes across the matrix is to push them on top of this branch :) I'll make separate PRs for what I can split.

Reply:
Move `project_urls` to separate PR and I can merge that now. I'm happy to have a broken link to `CI: Travis` for a few days.

Reply:
It won't be broken. The page exists, it just doesn't list builds :)